### ISSUE TITLE
The semantics represented by Fixpoint function

### ISSUE THREAD

**Issue by channgo2203** [pirapira/evmverif#6]
The semantics represented by Fixpoint function decreases on the number of step is not good. Since if a bytecode program uses jump instructions that loop, you cannot predict statically how many steps to evaluate. As my implementation (using fixpoint is "big-step" semantics style), it is decreases on the availabe gas.

**Comment by pirapira** [comment]
This is not a problem because whenever we use the fixpoint function, we can say "for any number of steps, the fixpoint function does not do this."  Or, "there exists a number of steps after which the fixpoint function does this".

**Comment by pirapira** [comment]
@channgo2203 My usage of `Fixpoint` function with `steps` comes from [here](https://www.cis.upenn.edu/~bcpierce/sf/current/ImpCEvalFun.html#lab356).  Eventually I will replace the steps with gas.

**Comment by channgo2203** [comment]
An other ways that is more flexible is to express the step evaluation as a relation instead of fixpoin function (you can look at some information at Software Foundations https://www.cis.upenn.edu/~bcpierce/sf/current/Smallstep.html https://www.cis.upenn.edu/~bcpierce/sf/current/Smallstep.html).

**Comment by pirapira** [comment]
@channgo2203 Yes.  That allows nondeterminism.  Actually we need nondeterminism for `CALL` instruction because the result of `CALL` is out of control of the contract under verification.  By default I'm going to wrap the currently existing `Fixpoint` function as a relation.
